collections/_members/bart-kalanski.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: person
+name: Bart Kalanski
+position: Contributor
+image_path:  https://pbs.twimg.com/profile_images/1271353670211375106/n4l3xu3k_400x400.jpg
+twitter: bartkodes
+github: https://github.com/bartkalanski
```

Comment:
This will just be `bartkalanski`

Reply:
Ohh sorry for that :)
I will commit the changes